src/main/java/org/kainos/ea/DropwizardTheITCrowdServiceApplication.java

```diff
@@ -21,16 +21,17 @@ public String getName() {
   @Override
   public void initialize(final Bootstrap<DropwizardTheITCrowdServiceConfiguration> bootstrap) {
     bootstrap.addBundle(new SwaggerBundle<DropwizardTheITCrowdServiceConfiguration>() {
+
       @Override
       protected SwaggerBundleConfiguration getSwaggerBundleConfiguration(DropwizardTheITCrowdServiceConfiguration configuration) {
-          return configuration.getSwagger();
+        return configuration.getSwagger();
       }
     });
   }
 
   @Override
-  public void run(final DropwizardTheITCrowdServiceConfiguration configuration,
-                  final Environment environment) {
+  public void run(final DropwizardTheITCrowdServiceConfiguration configuration, final Environment environment) {
     environment.jersey().register(new ConnectionController());
```

Comment:
You can remove everything related to `ConnectionController` and the test endpoint